mobile-privacy.html

```diff
@@ -1,16 +1,27 @@
 ---
 title: Privacy Policy
 ---
+<meta name="viewport" content="width=device-width">
 <style>
   .bc-content {
     font-family: 'Open Sans','Helvetica Neue', Helvetica, Arial, sans-serif;
     font-size: 1rem;
     line-height: 1.6;
     color: #4E5B6F;
+    padding: 0 10px;
+  }
+  .bc-content ul {
+    padding-left: 30px;
+  }
+  .bc-content ol > li > ul {
```

Comment:
you should apply ITCSS here. It would make your life easier, plus you avoid opening this kind of doors :)